docs/src/catalyst_applications/bifurcation_diagrams.md

```diff
@@ -1,115 +1,118 @@
-         :v => 2., :K => 20., :n => 3, :d => 0.05)
-bif_par = :S           # bifurcation parameter
-p_span = (0.1, 20.)    # interval to vary S over
-plot_var = :X          # we will plot X vs S
-nothing   # hide
+using BifurcationKit
+bif_par = :k1
+u_guess = [:X => 5.0, :Y => 2.0]
+p_start = [:k1 => 4.0, :k2 => 1.0, :k3 => 1.0, :k4 => 1.5, :k5 => 1.25]
+plot_var = :X
+bprob = BifurcationProblem(wilhelm_2009_model, u_guess, p_start, bif_par; plot_var=plot_var)
+nothing # hide
 ```
-When creating a bifurcation diagram, we typically start at some point in
-parameter-space. We will simply select the beginning of the interval over which
-we wish to compute the bifurcation diagram, `p_span[1]`. We thus create a
-modified parameter set where `S = 0.1`. For this parameter set, we also guess
-the steady state of the system. While a good estimate could be provided through
-an ODE simulation, BifurcationKit does not require the guess to be very
-accurate.
+
+BifurcationKit computes bifurcation diagrams using the `bifurcationdiagram` function. From an initial point in the diagram, it tracks the solution (using a continuation algorithm) until the entire diagram is computed (BifurcationKit's continuation can be used for other purposes, however, this tutorial focuses on bifurcation diagram computation). The continuation settings are provided in a `ContinuationPar` structure. In this example, we will only specify three settings, `p_min` and `p_max` (which sets the minimum and maximum values over which the bifurcation parameter is varied) and `max_steps` (the maximum number of continuation steps to take as the bifurcation diagram is tracked). We wish to compute a bifurcation diagram over the interval *(2.0,20.0)*, and will use the following settings:
 ```@example ex1
-p_bstart = copy(p)
-p_bstart[bif_par] = p_span[1]
-u0 = [:X => 1.0, :A => 1.0]
-nothing   # hide
+p_span = (2.0, 20.0)
+opts_br = ContinuationPar(p_min = p_span[1], p_max = p_span[2], max_steps=1000)
+nothing # hide
 ```
-Finally, we extract the ODE derivative function and its jacobian in a form that
-BifurcationKit can use:
+
+Finally, we compute our bifurcation diagram using:
 ```@example ex1
-oprob = ODEProblem(rn, u0, (0.0, 0.0), p_bstart; jac = true)
-F = (u,p) -> oprob.f(u, p, 0)
-J = (u,p) -> oprob.f.jac(u, p, 0)
-nothing   # hide
+bif_dia = bifurcationdiagram(bprob, PALC(), 2, (args...) -> opts_br; bothside=true)
+nothing # hide
 ```
+Where `PALC()` designates that we wish to use the pseudo arclength continuation method to track our solution. The third argument (`2`) designates the maximum number of recursions when branches of branches are computed (branches appear as continuation encounter certain bifurcation points). For diagrams with highly branches structure (rare for most standard chemical reaction networks) this input is important. Finally, `bothside=true` designates that we wish to perform continuation on both sides of the initial point (which is typically the case). 
```

Comment:
```suggestion
Where `PALC()` designates that we wish to use the pseudo arclength continuation method to track our solution. The third argument (`2`) designates the maximum number of recursions when branches of branches are computed (branches appear as continuation encounters certain bifurcation points). For diagrams with highly branched structures (rare for many common chemical reaction networks) this input is important. Finally, `bothside=true` designates that we wish to perform continuation on both sides of the initial point (which is typically the case). 
```

Reply:
Still want to use `most common` though. As in I think it is the case for the majority of CRNs, and this just suggest that one should not be surprised not to see a highly branched structure in a CRN (rather than that one should expect it, but correct me if I am wrong)